Inline Fallback Functions For Debugging

eellison opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #70656
* #70464
* #70871
* #70508
* __->__ #70463

Fix for https://github.com/pytorch/pytorch/issues/52940

When we call inlining on a fallback function, insert the runtime optimized version of its graph.

Differential Revision: [D33458651](https://our.internmc.facebook.com/intern/diff/D33458651)

Commit:
Optimized function inlining

[ghstack-poisoned]

jjsjann123 commented on torch/csrc/jit/passes/inliner.cpp:
```diff
@@ -30,8 +30,30 @@ void inlineCalls(Block* block) {
+              // optimized_graph() calls Inline, so we only need to explicitly
+              // invoke inlining on the jit optimized graph with recursive
+              // fallback funciton calls
+              Inline(*g.get());
```
:+1:

jjsjann123 reviewed (approved):
Debugging improvements! Awesome!

Commit:
Update on "Inline Fallback Functions For Debugging"


Fix for https://github.com/pytorch/pytorch/issues/52940

When we call inlining on a fallback function, insert the runtime optimized version of its graph.

[ghstack-poisoned]

Commit:
Update on "Inline Fallback Functions For Debugging"


Fix for https://github.com/pytorch/pytorch/issues/52940

When we call inlining on a fallback function, insert the runtime optimized version of its graph.

[ghstack-poisoned]

eellison commented:
@eellison has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33458651).

Commit:
Update on "Inline Fallback Functions For Debugging"


Fix for https://github.com/pytorch/pytorch/issues/52940

When we call inlining on a fallback function, insert the runtime optimized version of its graph.

Differential Revision: [D33458651](https://our.internmc.facebook.com/intern/diff/D33458651)

[ghstack-poisoned]

eellison commented:
@eellison has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33458651).

Commit:
Update on "Inline Fallback Functions For Debugging"


Fix for https://github.com/pytorch/pytorch/issues/52940

When we call inlining on a fallback function, insert the runtime optimized version of its graph.

Differential Revision: [D33458651](https://our.internmc.facebook.com/intern/diff/D33458651)

[ghstack-poisoned]

Commit:
Update on "Inline Fallback Functions For Debugging"


Fix for https://github.com/pytorch/pytorch/issues/52940

When we call inlining on a fallback function, insert the runtime optimized version of its graph.

Differential Revision: [D33458651](https://our.internmc.facebook.com/intern/diff/D33458651)

[ghstack-poisoned]

eellison commented:
@eellison has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33458651).
